README.md

```diff
@@ -1,4 +1,4 @@
-# {Name of the Project} - Your project should have a catchy and functional name
+# Citizen Air Quality Sensor
```

Comment:
Why was this changed in the pull request? I think you may have intended to change this in the main branch.

Reply:
That was a change for the main branch.  For some reason it changed all of them.  Do you have any suggestions for a more catchy name?

Reply:
Typically, it wouldn't change all the branches, it would only change the selected branch.